### ISSUE TITLE
Lack of attached gingiva

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11517]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 33005042 The gingiva is subdivided into the nonattached free gingiva (which includes the gingival margin and the papillae) and the attached gingiva. The attached gingiva is tightly and unmovably bound to the periosteum via collagen type I fibers. The gingiva is keratinized and thicker than the mucosa to provide mechanical protection. The border between attached gingiva and mucosa constitutes the mucogingival junction. In vivo, it is sometimes visible as a white line . The oral mucosa is loosely connected to the periosteum and nonkeratinized. It is movable, thin, more fragile, translucent, and blood vessels are visible.
(...) To distinguish periodontal EDS from generalized periodontitis affecting young individuals the diagnostic criteria of the 2017 classification of Ehlers–Danlos syndromes should be applied and recognition of complete **lack of attached gingiva** as a pathognomonic features in children and adults with pEDS should be recognized.

PMID: 41489249 Complete lack of attached gingiva is considered pathognomonic for pEDS and is  the most consistent clinical manifestation in children below the age of 10 years whereas the other features may be mild or absent during childhood.

**Parent term (use hpo.jax.org/app)**
Abnormality of the gingiva HP:0000168

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Periodontal Ehlers-Danlos syndrome  ORPHA:75392

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001467
- label: Lack of attached gingiva
- parents: [Abnormality of the gingiva (HP:0000168)](https://hpo.jax.org/app/browse/term/HP:0000168)
- pmid: PMID:33005042|PMID:41489249
- def: Absence of attached gingiva. The gingiva is subdivided into the nonattached free gingiva (which includes the gingival margin and the papillae) and the attached gingiva. The attached gingiva is tightly and unmovably bound to the periosteum via collagen type I fibers. The gingiva is keratinized and thicker than the mucosa to provide mechanical protection. The border between attached gingiva and mucosa constitutes the mucogingival junction. In vivo, it is sometimes visible as a white line . The oral mucosa is loosely connected to the periosteum and nonkeratinized. It is movable, thin, more fragile, translucent, and blood vessels are visible.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11517
- synonyms:
